database/migrations/2017_03_13_211740_CreateFriendsTable.php

```diff
@@ -14,6 +14,7 @@ class CreateFriendsTable extends Migration
     public function up()
     {
         Schema::create('friends', function (Blueprint $table) {
+            $table->increments('id');
```

Comment:
if we're adding id as primary key, we should now make the combination of username1 and username2 unique too